.vsconfig

```diff
@@ -0,0 +1,13 @@
+{
+  "version": "1.0",
+  "components": [

```

Comment:
Isn't there a Windows SDK we require?

Reply:
[RTF~M~ PR Description](https://github.com/microsoft/VFSForGit/pull/1143#issue-277754551) 😝 

> Note that the Windows 10 SDK 10240 is also required, but this component is no longer shipped with VS2019.

vsconfig only works in VS2019+, and the SDK we require 10240 is no longer included in the VS installation.

Reply:
sorry I missed that. the v141  tools are not included (by default) but can be downloaded this way. Can the windows SDK stuff download, too?

Reply:
No. The v141 VC toolchain is available in VS2019 (just not selected by default).
![image](https://user-images.githubusercontent.com/5658207/57539228-f57a1d00-7341-11e9-8b4b-d0ae3f3d3229.png)

The only available Windows SDKs in VS2019's installer are:
![image](https://user-images.githubusercontent.com/5658207/57539241-fd39c180-7341-11e9-9859-cb7b0c736e0c.png)

DevDiv have clearly decided to only support SDKs for currently supported Windows 10 versions. We must still support RS1 (apparently).